hoot-services/src/main/java/hoot/services/controllers/grail/GrailResource.java

```diff
@@ -403,11 +403,13 @@ public Response overpassSyncCheck(@Context HttpServletRequest request,
 
         String jobId = "grail_" + UUID.randomUUID().toString().replace("-", "");
         String id = DbUtils.getJobIdByTask(projectTaskInfo);
+        String bbox = DbUtils.getJobBbox(id);
 
         // Wait to detect overpass 'Last changeset pushed ID'
         GrailParams waitParams = new GrailParams();
         waitParams.setUser(user);
         waitParams.setTaskInfo(projectTaskInfo);
+        waitParams.setBounds(bbox);
```

Comment:
this let's you send a bbox param to overpass query?  was it not doing this before?  or just when the wait had timed out, the refresh wait was missing bbox?

Reply:
Only on resume